test/suites/smoke/test-dynamic-fees.ts

```diff
@@ -147,15 +167,48 @@ describeSuite({
         const receipts = (await apiAt.query.ethereum.currentReceipts()).unwrapOr([]);
         const events = await apiAt.query.system.events();
         const nextFeeMultiplier = await apiAt.query.transactionPayment.nextFeeMultiplier();
-        const fillPermill =
-          (weights.normal.refTime.unwrap().toBigInt() * 1_000_000n) /
-          apiAt.consts.system.blockWeights.perClass.normal.maxTotal.unwrap().refTime.toBigInt();
-        debug(`Block #${blockNum} fullness: ${(Number(fillPermill) / 10_000).toFixed(2)}%`);
+
+        // Get normal class weights
+        const blockWeights = apiAt.consts.system.blockWeights;
+        const normalMaxWeight = blockWeights.perClass.normal.maxTotal.unwrap();
+        const normalBlockWeight = weights.normal;
+
+        // Calculate normalized dimensions
+        const refTimeRatio =
+          (normalBlockWeight.refTime.unwrap().toBigInt() * 1_000_000_000n) /
+          (normalMaxWeight.refTime.unwrap().toBigInt() || 1n);
+        const normalizedRefTime = apiAt.registry.createType("Perbill", refTimeRatio.toString());
+
+        const proofSizeRatio =
+          (normalBlockWeight.proofSize.unwrap().toBigInt() * 1_000_000_000n) /
```

Comment:
We should add a comment explaining why we are multiplying by `1_000_000_000n`, for making the maintenance of this code easier.

Reply:
Also, we should probably use a constant.